PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -46,15 +46,53 @@ static bool isAllDigits(const generic_string &str)
 	return std::all_of(str.begin(), str.end(), ::isdigit);

```

Comment:
I never used it and moreover it's sample example code at [cpp reference](http://en.cppreference.com/w/cpp/string/wide/iswdigit) is bit confusing.